getting_started/developers/developers_cli.adoc

```diff
@@ -36,8 +36,8 @@ ifdef::openshift-online[]
 That said, similar to version 2, OpenShift provides out of the box a set of
 languages and link:../../using_images/db_images/index.html[databases] for
 developers with corresponding implementations and tutorials that allow you to
-kickstart your application development. Language support center around the
-link:../../dev_guide/templates.html#using-the-instantapp-templates[Instant App templates], 
+kickstart your application development. Language support centers around the
+link:../../dev_guide/app_tutorials/quickstarts.html#using-the-instantapp-templates[Quickstart templates], 
```

Comment:
Nope that anchor is wrong (I should have removed that anchor when I updated
the main link)

On Thu, Mar 10, 2016 at 11:05 AM, Ben Parees <notifications@github.com>
wrote:

> In getting_started/developers/developers_cli.adoc
> <https://github.com/openshift/openshift-docs/pull/1724#discussion_r55702148>
> :
>
> > @@ -36,8 +36,8 @@ ifdef::openshift-online[]
> >  That said, similar to version 2, OpenShift provides out of the box a set of
> >  languages and link:../../using_images/db_images/index.html[databases] for
> >  developers with corresponding implementations and tutorials that allow you to
> > -kickstart your application development. Language support center around the
> > -link:../../dev_guide/templates.html#using-the-instantapp-templates[Instant App templates],
> > +kickstart your application development. Language support centers around the
> > +link:../../dev_guide/app_tutorials/quickstarts.html#using-the-instantapp-templates[Quickstart templates],
>
> Is that anchor right? It still says instant app. If it is right, can you
> change the actual anchor and then update this?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/openshift-docs/pull/1724/files#r55702148>.
>